docs/administration/managing-users-and-teams/system-and-space-permissions.md

```diff
@@ -0,0 +1,44 @@
+---
+title: System and Space Permissions
+description: An explanation of the two levels that permissions can apply at, the Space and the System.
+---
+
+Octopus Deploy supports partitioning your server up into [Spaces](../spaces) which enables teams to stay focused on only the projects and content that matter to those teams. As a result, permission scoping needs to respect boundaries that support both the administration of the whole server, as well as support the administration of an individual space.
+
+This introduces some complexity that can be useful to understand when things don't work quite the way you expect. Reading this page should give you a general understanding of how permissions work in these two contexts. 
+
+## Levels of permission
+While designing this feature, we needed to reason about which API resources would need to be configured _outside_ of a space and which resources should _only_ be configured _within_ a space. 
+
+That means that when considering permissions, we need to think in terms of the two administrative use cases of an Octopus Deploy instance - administering the system itself, and administering a space. Since these are very different things, permissions need to be considered as applying at these two 'levels': the **System** and **Space** levels.  
+
+These levels are in fact attached to the nature of the API resources themselves, if a resource is considered 'space only' then permissions required to access that resource are considered to be space level permissions.
+
+When you design or inspect your own custom **user roles**, we present this information to help you reason about the types of permissions you are granting that role, so that you are able to appropriately restrict access to the various resources that you care about.
+
+### What is a 'System level' permission?
+**System** level permissions are those that involve administering the entire system, but do not include permissions within an individual Space. An example of system level permissions are the **User** permissions, since users are not scoped to a space.
+
+In practical terms, this means that a resource within a space, or another system level resource, may reference the users configured in the system. 
```

Comment:
I think this is unrelated to permissions and might be confusing. Let's remove this.